services/user.py

```diff
@@ -0,0 +1,49 @@
+from db.models import User
```

Comment:
Consider using `get_user_model()` instead of directly importing `User` for better flexibility and adherence to Django best practices.